inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -1281,6 +1281,23 @@ impl_types layout_optimizer::get_preferred_impl_type(program_node& node, format
             impl_candidate = impl_types::ocl;
         }
 
+        // [WA] to resolve onednn kernel issue of multiple sum post-ops
```

Comment:
I think the commit title is misleading.
It is not about `supporting`, but about `avoiding`.

Reply:
Fixed